src/BlockTypes/ProductGalleryLargeImage.php

```diff
@@ -120,7 +120,7 @@ private function get_main_images_html( $context, $product_id ) {
 			'data-wc-bind--style' => 'selectors.woocommerce.productGalleryLargeImage.styles',
 			'data-wc-effect'      => 'effects.woocommerce.scrollInto',
 			'class'               => 'wc-block-woocommerce-product-gallery-large-image__image',
-
+			'data-wc-class--wc-block-woocommerce-product-gallery-large-image__image--active-image-slide' => 'selectors.woocommerce.isSelected',
```

Comment:
It seems this was just added to be able to locate the container? Couldn't we just use what's already there for the class name? https://github.com/woocommerce/woocommerce-blocks/pull/11536/files#diff-1ad160860b7b0cdac44eab7d3344949822d8d32db90384d2ecd92c9fcb1700d3R122

Reply:
After the animation was added to the Large Image block, all the images are available when the page loads. The reason they do not display all at once is because we set the overflow:hidden property on them with CSS. Currently, there is no way to define which one is the active image when it becomes visible on the screen because all of them are in the HTML, and none of them are in a hidden state (which prevent us from using the 'visible' property on Playwright to select the correct one).

For the Product Image Gallery block (the previous block), they control the active image by setting an `...-active-slide` class on the selected image. So, I borrowed that idea from there 😅 .